tests/runtimes/local/app/utils_test.py

```diff
@@ -0,0 +1,29 @@
+"""Unit tests for the import_scan module."""
+
+from ostorlab.runtimes.local.app import utils
+from ostorlab.runtimes.local.models import models
+from ostorlab.utils import risk_rating
+
+
+def testImportScan_always_shouldImportScan(
```

Comment:
I don't see any unit test related to the graphql endpoint.

Reply:
added, please check again